src/gwt/src/org/rstudio/studio/client/workbench/views/help/Help.java

```diff
@@ -274,6 +274,16 @@ void onOpenSparklyrCheatSheet()
       openCheatSheet("sparklyr_cheat_sheet");
    }
    
+   void onOpenPurrrCheatSheet()
+   {
+      openCheatSheet("purrr_cheat_sheet");
+   }
+   
+   void onBrowseCheatSheets()
+   {
+      globalDisplay_.openWindow("https://www.rstudio.com/resources/cheatsheets/");
```

Comment:
Should we omit the protocol here? (so that `http` vs. `https` can be automatically chosen?)

Reply:
IIRC we do need a fully qualified URL here because on the desktop the operating system's URL handler is invoked and it might not work properly without a protocol.

Reply:
see e.g. here where we explicitly require a protocol:

https://github.com/rstudio/rstudio/blob/17ba3173100489938a99ead4d36f57e0ef65a40b/src/gwt/src/org/rstudio/studio/client/common/impl/DesktopWindowOpener.java#L33